src/core/Link/Link/Link.md

```diff
@@ -1,25 +1,39 @@
 ```js
 import { Link, Paragraph } from 'suomifi-ui-components';
-
-<Paragraph>
+<>
   <Link
     className="test-classname"
     href="https://www.notvisitedlink.com/"
     underline="initial"
   >
     Not visited link
-  </Link>{' '}
+  </Link>
+
   <Link className="test-classname" href="#" underline="initial">
     Visited link
-  </Link>{' '}
+  </Link>
+
+  <Link href="https://www.notvisitedlink.com/" underline="hover">
+    Link without underline
+  </Link>
+
   <Link
-    className="test-classname"
+    variant="accent"
     href="https://www.notvisitedlink.com/"
-    underline="hover"
+    underline="initial"
```

Comment:
`underline` prop does not seem to do anything here?

Reply:
It was overridden by the accent link styles. I changed it so that the prop is now applied, but removed it from the examples.